rubric.md

```diff
@@ -0,0 +1,30 @@
+|---------------------------------------------------------------------------------------------------------|--------------------------------|------------------------|--------------------------|
+| Technical Requirements                                                                                  | Does Not Meet Expectations (0) | Meets Expectations (1) | Exceeds Expectations (2) |
+| Project Proposal / Wireframes turned in week 5                                                          |                                |                        |                          |
+| Draft of HTML/CSS turned in week 7                                                                      |                                |                        |                          |
+| First draft of JS turned in week 8                                                                      |                                |                        |                          |
+| Updated rough draft of project week 9                                                                   |                                |                        |                          |
+| Final presentation                                                                                      |                                |                        |                          |
+| Use HTML5 structural elements (header, footer, nav, footer)                                             |                                |                        |                          |
+| Demonstrate a correct use of classes and IDs                                                            |                                |                        |                          |
+| Use classes to identify a type of element and IDs for identifying a specific unique element on the page |                                |                        |                          |
+| Select the appropriate tags to markup content                                                           |                                |                        |                          |
+| Apply fonts, color and styles to elements and the page                                                  |                                |                        |                          |
+| Demonstrate use of floats (or inline-block) and the box model                                           |                                |                        |                          |
+| Use JavaScript/jQuery to make pages interactive                                                         |                                |                        |                          |
+| TOTAL: ______/24                                                                                        |                                |                        |                          |
```

Comment:
Overall nice job. There are some fundamentals to clean up but overall you've got a cool design idea and a chance to play around. Make sure to clean up any unused CSS, JS or HTML. If an element isn't being referenced or styled, ask yourself "Does it need to be here?" It'll make your site easier to maintain and update in the long run.

This is ready to merge whenever you are!